src/consts.js

```diff
@@ -212,7 +212,7 @@ export const ACTOR_LIMITS = {
     RUN_DISK_TO_MEMORY_SIZE_COEFF: 2,
 
     // For each build or run container, set CPU cores based on memory size
-    RUN_MEMORY_SIZE_TO_CPU_CORES_COEFF: 4096,
```

Comment:
Are you sure that this is not used anywhere? (web, docs, sdk, ... )?

Reply:
That would mean the change was faster than light :) See files changed

Reply:
I'm sure, I added this const in the previous commit in this PR, and then just renamed it to another name per Honza's suggestion (and I've renamed it to the new name in the related PR on Bitbucket as well, which was the only place where it was used so far).